hack/make.sh

```diff
@@ -29,11 +29,22 @@ export MAKEDIR="$SCRIPTDIR/make"
 
 # We're a nice, sexy, little shell script, and people might try to run us;
 # but really, they shouldn't. We want to be in a container!
-if [ "$PWD" != "/go/src/$DOCKER_PKG" ] || [ -z "$DOCKER_CROSSPLATFORMS" ]; then
+INCONTAINER=1
```

Comment:
Can we switch this to `inContainer=` to match other uses of local variables within the scripts?  We try to make only envs or cross-script vars all-caps.